tests/unit/cli/test_download.py

```
@@ -893,3 +893,664 @@ def test_download_name_not_found(self, runner, mock_auth, mock_fetch_tokens):
         assert result.exit_code != 0
         # Should mention no match found or available artifacts
         assert "No artifact" in result.output or "nonexistent" in result.output.lower()
+
+
+# =============================================================================
+# DOWNLOAD QUIZ + FLASHCARDS STANDARD-FLAG TESTS (P2.T4)
+# =============================================================================
+
+
+def make_quiz_artifact(
+    id: str, title: str, status: int = 3, created_at: datetime | None = None
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)